forge-gui/res/cardsfolder/upcoming/fear_fire_foes.txt

```diff
@@ -0,0 +1,10 @@
+Name:Fear, Fire, Foes!
+ManaCost:X R
+Types:Sorcery
+A:SP$ Effect | StaticAbilities$ STCantPrevent | AILogic$ Burn | SubAbility$ DBDealDamage | SpellDescription$ CARDNAME deals X damage to target creature and 1 damage to each other creature with the same controller.
+SVar:DBDealDamage:DB$ DealDamage | ValidTgts$ Creature | NumDmg$ X | NoPrevention$ True | RememberTargets$ True | SubAbility$ DBDamageAll 
+SVar:DBDamageAll:DB$ DamageAll | ValidCards$ Creature.IsNotRemembered+ControlledBy TargetedOrController | NumDmg$ 1 | SubAbility$ DBCleanup
```

Comment:
this damage should be dealt simultaneously

Reply:
Damagemap prevents the damage ?

Reply:
no it's needed

Reply:
that's not how you use it
better look at cards like _Angrath, Minotaur Pirate_ if you don't know

Reply:
Can't figure out how to prevent the one damage to not be dealt to targeted creature :(

Reply:
Why did you remove the RememberTargets$ part?
that should have worked

Reply:
I've restored it but It doesn't work

Reply:
Ok I'll check

Reply:
Only works on the top SA
but try this, should be even easier:
`NotDefinedTargeted`

Reply:
Thanks seems to work